src/fortran_routing/Reservoir_Binding/Reservoirs/makefile

```diff
@@ -1,8 +1,8 @@
 NETCDFINC := $(shell nc-config --fflags)
 NETCDFLIB := $(shell nc-config --flibs)
-COMPILER90 = $(F90)
-F90FLAGS := -g -w -c -ffree-form -ffree-line-length-none -fconvert=big-endian -frecord-marker=4 $(NETCDFINC)
-F90LFLAGS := $(NETCDFLIB) -static-libgfortran -static-libgcc -nodefaultlibs -lgfortran -lgcc -fPIC
+FC = $(F90)
+FFLAGS += -g -w -c -ffree-form -ffree-line-length-none -fconvert=big-endian -frecord-marker=4 $(NETCDFINC)
```

Comment:
Why += here?  `:=` is an immediate assignment, which takes place in the first pass parse of the makefile and all references are replaced with those contents.  `+=` is likely to result in a recursive, deffered variable that has to be re-evaluated for each refernce.

Reply:
Good question. The only answer that I can give is this, where I found the suggestion to use `+=`: 
https://www.gnu.org/software/make/manual/html_node/Appending.html

I'm open to either, especially given that it appears that the `+=` is mostly for specialized cases. But it does seem to be the preferred/recommended method in that article.

Reply:
Another reference ... again, I'm having a hard time teasing a solid recommendation one way or the other from this, so I left it with "this works" on the version in the merged code.
https://stackoverflow.com/questions/35072192/how-to-make-make-append-additional-standard-flags